tests/jballerina-unit-test/src/test/resources/test-src/expressions/access/field_access.bal

```diff
@@ -14,6 +14,8 @@
 // specific language governing permissions and limitations
 // under the License.
 
+import ballerina/test;
```

Comment:
We don't usually import non-langlib modules for lang tests. We can introduce a simple langlib function here for assertion.